Mathlib/Data/Set/Card.lean

```diff
@@ -927,6 +927,16 @@ theorem eq_univ_iff_ncard [Finite α] (s : Set α) :
     s = univ ↔ ncard s = Nat.card α := by
   rw [← compl_empty_iff, ← ncard_eq_zero, ← ncard_add_ncard_compl s, left_eq_add]
 
+lemma even_ncard_compl_iff [Fintype α] (heven : Even (Fintype.card α)) (s : Set α) :
+    Even sᶜ.ncard ↔ Even s.ncard := by
```

Comment:
I think it's a little unconventional to mix Fintype and Finite like this; can we instead have inputs `[Finite α] (heven : Even (Nat.card α))` (and outputs the same)? Usually we'd have `Finite/Nat.card/Set.ncard` or `Fintype/Fintype.card/Finset.card`, but not a mix of both (except for lemmas whose job is precisely to translate between them)

Reply:
Adopted the first version:)

Reply:
I'm still not convinced of this - do you mind if I push a change?

Reply:
I certainly don't mind, go ahead:)

Reply:
@b-mehta My apologies, I had some local changes lost in the mix. It should be better now, but feel free to adapt it more if you think there's something to gain.

Reply:
Ah, the last batch of changes indeed resolve this, thanks!